pandas/tests/test_tile.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+from pandas import DataFrame, Series
+import pandas.util.testing as tm
+
+from pandas import cut
+
+
+class TestCut(object):
+
```

Comment:
don't add a new file, put in pandas/tests/reshaping/test_tile.py

Reply:
Sorry, I didn't find it before.